packages/one-app-server-bundler/utils/patchedCryptoHash.js

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2023 American Express Travel Related Services Company, Inc.
+ * Copyright 2024 American Express Travel Related Services Company, Inc.
```

Comment:
This file shouldn't be needed with the upgrade. It was a limitation of webpack v4